mobile/lib/common/recover_dlc_change_notifier.dart

```diff
@@ -0,0 +1,55 @@
+import 'package:f_logs/model/flog/flog.dart';
+import 'package:flutter/material.dart';
+import 'package:get_10101/bridge_generated/bridge_definitions.dart' as bridge;
+import 'package:get_10101/common/application/event_service.dart';
+import 'package:get_10101/common/domain/background_task.dart';
+import 'package:get_10101/common/global_keys.dart';
+import 'package:get_10101/features/trade/order_submission_status_dialog.dart';
+import 'package:provider/provider.dart';
+
+class RecoverDlcChangeNotifier extends ChangeNotifier implements Subscriber {
+  late TaskStatus taskStatus;
+
+  @override
+  void notify(bridge.Event event) {
+    if (event is bridge.Event_BackgroundNotification) {
+      if (event.field0 is! bridge.BackgroundTask_RecoverDlc) {
+        // ignoring other kinds of background tasks
+        return;
+      }
+      RecoverDlc recoverDlc = RecoverDlc.fromApi(event.field0 as bridge.BackgroundTask_RecoverDlc);
+      FLog.debug(text: "Received a recover dlc event. Status: ${recoverDlc.taskStatus}");
+
+      taskStatus = recoverDlc.taskStatus;
+
+      if (taskStatus == TaskStatus.pending) {
+        // initialize dialog for the pending task
+        showDialog(
+          context: shellNavigatorKey.currentContext!,
+          builder: (context) {
+            TaskStatus status = context.watch<RecoverDlcChangeNotifier>().taskStatus;
+
+            // todo(holzeis): Reusing the order submission status dialog is not nice, but it's actually suitable for any task execution that has pending,
```

Comment:
is there a particular reason you're not just keeping this as normal `TODO`s ? e.g. do you have a commit hook that checks for these specifically? 

on my end is worse than normal `TODO` as it's harder to spot it both with the naked eye when reading the code and also doesn't get picked up by my IDE which e.g. automatically shows me TODOs that are not on the main branch.

also, one can see the author of a TODO with regular `git blame`.

if you don't mind either way, please consider the points above 🙏

Reply:
I think I've seen it on other projects (`rust-dlc`?) done like that - and I like it. It shows ownership of the todo. `git blame` is not always trustworthy as moving code will easily change that.

I like doing it like that and I don't think it hurts, but if you are feeling strongly about it I can change it to a regular todo.

fyi: my IDE is highlighting these todos the same way its highlighting any other todo.

Reply:
if you are using this plugin https://github.com/alphapapa/magit-todos, I guess the only issue was that I did not capitalized the keyword.

> Items in KEYWORD(username): format are also read

I have capitalized all fixmes and todos, let me know if this is working for you now.